baseplate/core.py

```diff
@@ -187,6 +190,7 @@ def configure_tracing(self, service_name,
                 num_span_workers=num_span_workers,
                 span_batch_interval=span_batch_interval,
                 sample_rate=sample_rate,
+                log_if_unconfigured=log_if_unconfigured,
```

Comment:
I don't feel strongly either way; I hadn't anticipated so many options being needed so I can definitely see the case for wrapping the config parsing into an object. Both seem to afford the same flexibility in configuration. This would break existing client integrations though right?